st2common/tests/unit/test_spec_loader.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2020 The StackStorm Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+
+import unittest2
+import yaml
+
+from st2common.util import spec_loader
+
+
+class SpecLoaderTest(unittest2.TestCase):
```

Comment:
Also wouldn't hurt to add a test case which verifies SafeLoad is indeed used, but otherwise, LGTM.